tools/tests/apps/modules/tests.js

```diff
@@ -99,11 +99,6 @@ describe("app modules", () => {
     }
   });
 
-  it("should not be parsed in strictMode", () => {
```

Comment:
This indicates a bug in reify. We should keep this test and fix reify. There was a discussion on slack about it: https://meteor-community.slack.com/archives/C0669FW38UU/p1710181750813509

Reply:
I see. I'll have a look at this later. Thanks.

Reply:
Thanking about this more, if we want to switch modules to being in strict mode this might be a good time to do it. Modules are supposed to be in strict mode, so if we are fine with the breaking changes then we could leave reify as is.

Reply:
I agree with you. I also think it's the right time to do this time of change. I'm going to merge this PR, and we'll consider this a breaking change.